server/executor/trace_poller.go

```diff
@@ -123,7 +123,7 @@ func (tp tracePoller) Stop() {
 func (tp tracePoller) Poll(ctx context.Context, test model.Test, run model.Run) {
 	log.Printf("[TracePoller] Test %s Run %d: Poll\n", test.ID, run.ID)
 
-	job := NewPollingRequest(ctx, test, run, 0)
+	job := NewPollingRequest(ctx, test, run, 1)
```

Comment:
Can we put this number as a constant on this file? Something like `PollingRequestStartIteration` ?

```suggestion
	job := NewPollingRequest(ctx, test, run, PollingRequestStartIteration)
```